EligibilityCheckOrchestrator/index.ts

```diff
@@ -72,25 +90,20 @@ const EligibilityCheckOrchestrator = df.orchestrator(function*(
     addSeconds(context.df.currentUtcDateTime, NOTIFICATION_DELAY_SECONDS)
   );
 
-  const eligibilityCheck = toEligibilityCheckFromDSU(
-    eligibilityCheckResponse.data,
-    eligibilityCheckResponse.fiscalCode,
-    eligibilityCheckResponse.validBefore
-  );
-
   // send push notification with eligibility details
   yield context.df.callActivity(
     "SendMessageActivity",
-    EligibilityCheckFailure.is(eligibilityCheck)
+    // TODO: Handle notification for EligibilityCheckSuccessConflict
+    EligibilityCheckFailure.is(validatedEligibilityCheck)
```

Comment:
I sugget to refactor this part into a new method that return the correct message switching on EligibilityCheck status and returns "none" in case there's nothing (to avoid calling the activity with {}) ie.

```
getMessage(validatedEligibilityCheck).map(msg =>
                     yield context.df.callActivity("SendMessageActivity", msg))
```

Reply:
done here: https://github.com/pagopa/io-functions-bonus/pull/56